types/json-ptr/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "baseUrl": "../",
+        "esModuleInterop": true,
```

Comment:
this needs to be false to test backward compatibility

Reply:
Done. 🙂